geniza/annotations/views.py

```diff
@@ -30,6 +31,38 @@ class ApiAccessMixin(AccessMixin):
     raise_exception = True  # return an error instead of redirecting to login
 
 
+class AnnotationLastModifiedMixin(View):
```

Comment:
was a little surprised you need a whole mixin for this, but I guess it's nice to have the conditional logic encapsulated in one place and separate from the main annotation detail view

Reply:
It may not be necessary; was going off the initial idea [here](https://github.com/Princeton-CDH/geniza/issues/965#issue-1311261872) but I guess it sort of lost the generic-ness and became about annotations… I suppose the only way to make the mixin generic would be the dispatch function alone, which is pretty small.